### ISSUE TITLE
[BUG] Android softreboot with system crash visible in the logs

### ISSUE THREAD

**Issue by Tutul-** [ElderDrivers/EdXposed#620]
**What happened?**

Since the last update of my ROM, the system soft reboot often (from a couple of minutes to ~15 minutes).
At the same time I've updated Nanodroid (only with microg+fdroid+google sync and nothing else) so I'm not sure if it's the ROM update or microg in any way.

I tested with SandHook and YAHFA and both cause the crash.

**Xposed Module List**

Greenify and NoQSInLockScreen (EdXposed module is disabled)

**Magisk Module List**

Busybox
DNSCrypt-proxy2
Nanodroid (microg/google/fdroid/patcher)
Riru Core

**Versions of EdXposed and Riru**

EdXposed: v4.5.7 with v0.5.0.6 alpha installed since it's aivalable

Riru: v21.3

**Logcat**

YAHFA
```
--------- beginning of crash
10-27 21:18:37.407  2012  2171 F libc    : Fatal signal 6 (SIGABRT), code -1 (SI_QUEUE) in tid 2171 (Binder:2012_2), pid 2012 (system_server)
10-27 21:18:37.609 13649 13649 F DEBUG   : *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***
10-27 21:18:37.610 13649 13649 F DEBUG   : LineageOS Version: '17.1-20201027-NIGHTLY-lake'
10-27 21:18:37.610 13649 13649 F DEBUG   : Build fingerprint: 'motorola/lake_retail/lake:10/QPW30.61-18/d18ed:user/release-keys'
10-27 21:18:37.610 13649 13649 F DEBUG   : Revision: 'pvt'
10-27 21:18:37.610 13649 13649 F DEBUG   : ABI: 'arm64'
10-27 21:18:37.611 13649 13649 F DEBUG   : Timestamp: 2020-10-27 21:18:37+0100
10-27 21:18:37.611 13649 13649 F DEBUG   : pid: 2012, tid: 2171, name: Binder:2012_2  >>> system_server <<<
10-27 21:18:37.611 13649 13649 F DEBUG   : uid: 1000
10-27 21:18:37.611 13649 13649 F DEBUG   : signal 6 (SIGABRT), code -1 (SI_QUEUE), fault addr --------
10-27 21:18:37.611 13649 13649 F DEBUG   : Abort message: 'No inline cache found for void com.android.server.wm.-$$Lambda$SurfaceAnimationRunner$puhYAP5tF0mSSJva-eUz59HnrkA.onAnimationUpdate(android.animation.ValueAnimator)@40'
10-27 21:18:37.611 13649 13649 F DEBUG   :     x0  0000000000000000  x1  000000000000087b  x2  0000000000000006  x3  0000007725318e20
10-27 21:18:37.611 13649 13649 F DEBUG   :     x4  00000077253188c8  x5  00000077253188c8  x6  00000077253188c8  x7  0000000000000000
10-27 21:18:37.611 13649 13649 F DEBUG   :     x8  00000000000000f0  x9  05fcf5b29bcc18ca  x10 0000000000000001  x11 0000000000000000
10-27 21:18:37.611 13649 13649 F DEBUG   :     x12 fffffff0fffffbdf  x13 0000000010f88910  x14 0000000000000068  x15 055784e0150cc643
10-27 21:18:37.611 13649 13649 F DEBUG   :     x16 00000077b89d58c0  x17 00000077b89b33b0  x18 00000076cfb06000  x19 00000000000007dc
10-27 21:18:37.611 13649 13649 F DEBUG   :     x20 000000000000087b  x21 00000000ffffffff  x22 00000077269d9fc0  x23 0000007735836000
10-27 21:18:37.611 13649 13649 F DEBUG   :     x24 00000077353052e7  x25 0000007735838000  x26 000000773523b258  x27 0000007735838000
10-27 21:18:37.611 13649 13649 F DEBUG   :     x28 0000000000000000  x29 0000007725318ec0
10-27 21:18:37.611 13649 13649 F DEBUG   :     sp  0000007725318e00  lr  00000077b8966f48  pc  00000077b8966f74
10-27 21:18:38.187 13649 13649 F DEBUG   : 
10-27 21:18:38.187 13649 13649 F DEBUG   : backtrace:
10-27 21:18:38.187 13649 13649 F DEBUG   :       #00 pc 0000000000081f74  /apex/com.android.runtime/lib64/bionic/libc.so (abort+160) (BuildId: bf14cf7a62d1f91755beddd4a937354d)
10-27 21:18:38.187 13649 13649 F DEBUG   :       #01 pc 00000000004b14a4  /apex/com.android.runtime/lib64/libart.so!libart.so (offset 0x4b1000) (art::Runtime::Abort(char const*)+2268) (BuildId: 77524e83b39d990eaaf7c2af505c5eae)
10-27 21:18:38.187 13649 13649 F DEBUG   :       #02 pc 000000000000c5b4  /system/lib64/libbase.so (android::base::LogMessage::~LogMessage()+608) (BuildId: 85c53926c6f8d70f35d67c6e99bae271)
10-27 21:18:38.187 13649 13649 F DEBUG   :       #03 pc 0000000000342778  /apex/com.android.runtime/lib64/libart.so!libart.so (offset 0x292000) (art::ProfilingInfo::GetInlineCache(unsigned int)+272) (BuildId: 77524e83b39d990eaaf7c2af505c5eae)
10-27 21:18:38.187 13649 13649 F DEBUG   :       #04 pc 00000000003427c8  /apex/com.android.runtime/lib64/libart.so!libart.so (offset 0x292000) (art::ProfilingInfo::AddInvokeInfo(unsigned int, art::mirror::Class*)+28) (BuildId: 77524e83b39d990eaaf7c2af505c5eae)
10-27 21:18:38.187 13649 13649 F DEBUG   :       #05 pc 000000000059ace4  /apex/com.android.runtime/lib64/libart.so!libart.so (offset 0x4b3000) (MterpInvokeVirtualRange+444) (BuildId: 77524e83b39d990eaaf7c2af505c5eae)
10-27 21:18:38.187 13649 13649 F DEBUG   :       #06 pc 0000000000130b14  /apex/com.android.runtime/lib64/libart.so (mterp_op_invoke_virtual_range+20) (BuildId: 77524e83b39d990eaaf7c2af505c5eae)
10-27 21:18:38.187 13649 13649 F DEBUG   :       #07 pc 00000000001eaaec  [anon:dalvik-classes4.dex extracted in memory from /system/framework/services.jar!classes4.dex] (com.android.server.am.ActivityManagerService.broadcastIntentLocked+80)
10-27 21:18:38.187 13649 13649 F DEBUG   :       #08 pc 000000000059d470  /apex/com.android.runtime/lib64/libart.so!libart.so (offset 0x4b3000) (MterpInvokeStaticRange+768) (BuildId: 77524e83b39d990eaaf7c2af505c5eae)
10-27 21:18:38.187 13649 13649 F DEBUG   :       #09 pc 0000000000130c94  /apex/com.android.runtime/lib64/libart.so (mterp_op_invoke_static_range+20) (BuildId: 77524e83b39d990eaaf7c2af505c5eae)
10-27 21:18:38.187 13649 13649 F DEBUG   :       #10 pc 0000000000000722  [anon:dalvik-DEX data] (EdHooker_.hook+858)
10-27 21:18:38.187 13649 13649 F DEBUG   :       #11 pc 00000000002afd20  /apex/com.android.runtime/lib64/libart.so!libart.so (offset 0x292000) (_ZN3art11interpreterL7ExecuteEPNS_6ThreadERKNS_20CodeItemDataAccessorERNS_11ShadowFrameENS_6JValueEbb.llvm.1271440803783865717+240) (BuildId: 77524e83b39d990eaaf7c2af505c5eae)
10-27 21:18:38.187 13649 13649 F DEBUG   :       #12 pc 0000000000588e8c  /apex/com.android.runtime/lib64/libart.so!libart.so (offset 0x4b3000) (artQuickToInterpreterBridge+1012) (BuildId: 77524e83b39d990eaaf7c2af505c5eae)
10-27 21:18:38.187 13649 13649 F DEBUG   :       #13 pc 000000000013f468  /apex/com.android.runtime/lib64/libart.so (art_quick_to_interpreter_bridge+88) (BuildId: 77524e83b39d990eaaf7c2af505c5eae)
10-27 21:18:38.187 13649 13649 F DEBUG   :       #14 pc 000000000205c7e8  /memfd:/jit-cache (deleted) (com.android.server.am.ActivityManagerService.broadcastIntent+424)
10-27 21:18:38.187 13649 13649 F DEBUG   :       #15 pc 00000000004e6590  /system/framework/arm64/boot-framework.oat (android.app.ContextImpl.sendStickyBroadcastAsUser+352) (BuildId: 268218f903f195e6053515289a297317303f358b)
10-27 21:18:38.187 13649 13649 F DEBUG   :       #16 pc 0000000000136334  /apex/com.android.runtime/lib64/libart.so (art_quick_invoke_stub+548) (BuildId: 77524e83b39d990eaaf7c2af505c5eae)
10-27 21:18:38.187 13649 13649 F DEBUG   :       #17 pc 000000000014506c  /apex/com.android.runtime/lib64/libart.so (art::ArtMethod::Invoke(art::Thread*, unsigned int*, unsigned int, art::JValue*, char const*)+244) (BuildId: 77524e83b39d990eaaf7c2af505c5eae)
10-27 21:18:38.188 13649 13649 F DEBUG   :       #18 pc 00000000002df0d4  /apex/com.android.runtime/lib64/libart.so!libart.so (offset 0x292000) (art::interpreter::ArtInterpreterToCompiledCodeBridge(art::Thread*, art::ArtMethod*, art::ShadowFrame*, unsigned short, art::JValue*)+384) (BuildId: 77524e83b39d990eaaf7c2af505c5eae)
10-27 21:18:38.188 13649 13649 F DEBUG   :       #19 pc 00000000002da3b4  /apex/com.android.runtime/lib64/libart.so!libart.so (offset 0x292000) (bool art::interpreter::DoCall<false, false>(art::ArtMethod*, art::Thread*, art::ShadowFrame&, art::Instruction const*, unsigned short, art::JValue*)+912) (BuildId: 77524e83b39d990eaaf7c2af505c5eae)
10-27 21:18:38.188 13649 13649 F DEBUG   :       #20 pc 0000000000597844  /apex/com.android.runtime/lib64/libart.so!libart.so (offset 0x4b3000) (MterpInvokeVirtual+648) (BuildId: 77524e83b39d990eaaf7c2af505c5eae)
10-27 21:18:38.188 13649 13649 F DEBUG   :       #21 pc 0000000000130814  /apex/com.android.runtime/lib64/libart.so (mterp_op_invoke_virtual+20) (BuildId: 77524e83b39d990eaaf7c2af505c5eae)
10-27 21:18:38.188 13649 13649 F DEBUG   :       #22 pc 00000000001cd9cc  [anon:dalvik-classes4.dex extracted in memory from /system/framework/services.jar!classes4.dex] (com.android.server.TelephonyRegistry.broadcastSignalStrengthChanged+108)
10-27 21:18:38.188 13649 13649 F DEBUG   :       #23 pc 00000000002afd20  /apex/com.android.runtime/lib64/libart.so!libart.so (offset 0x292000) (_ZN3art11interpreterL7ExecuteEPNS_6ThreadERKNS_20CodeItemDataAccessorERNS_11ShadowFrameENS_6JValueEbb.llvm.1271440803783865717+240) (BuildId: 77524e83b39d990eaaf7c2af505c5eae)
10-27 21:18:38.188 13649 13649 F DEBUG   :       #24 pc 0000000000588e8c  /apex/com.android.runtime/lib64/libart.so!libart.so (offset 0x4b3000) (artQuickToInterpreterBridge+1012) (BuildId: 77524e83b39d990eaaf7c2af505c5eae)
10-27 21:18:38.188 13649 13649 F DEBUG   :       #25 pc 000000000013f468  /apex/com.android.runtime/lib64/libart.so (art_quick_to_interpreter_bridge+88) (BuildId: 77524e83b39d990eaaf7c2af505c5eae)
10-27 21:18:38.188 13649 13649 F DEBUG   :       #26 pc 00000000020a276c  /memfd:/jit-cache (deleted) (com.android.server.TelephonyRegistry.notifySignalStrengthForPhoneId+1196)
10-27 21:18:38.188 13649 13649 F DEBUG   :       #27 pc 0000000000b47614  /system/framework/arm64/boot-framework.oat (com.android.internal.telephony.ITelephonyRegistry$Stub.onTransact+6820) (BuildId: 268218f903f195e6053515289a297317303f358b)
10-27 21:18:38.188 13649 13649 F DEBUG   :       #28 pc 0000000000839160  /system/framework/arm64/boot-framework.oat (android.os.Binder.execTransactInternal+704) (BuildId: 268218f903f195e6053515289a297317303f358b)
10-27 21:18:38.188 13649 13649 F DEBUG   :       #29 pc 0000000000838d78  /system/framework/arm64/boot-framework.oat (android.os.Binder.execTransact+296) (BuildId: 268218f903f195e6053515289a297317303f358b)
10-27 21:18:38.188 13649 13649 F DEBUG   :       #30 pc 0000000000136334  /apex/com.android.runtime/lib64/libart.so (art_quick_invoke_stub+548) (BuildId: 77524e83b39d990eaaf7c2af505c5eae)
10-27 21:18:38.188 13649 13649 F DEBUG   :       #31 pc 000000000014506c  /apex/com.android.runtime/lib64/libart.so (art::ArtMethod::Invoke(art::Thread*, unsigned int*, unsigned int, art::JValue*, char const*)+244) (BuildId: 77524e83b39d990eaaf7c2af505c5eae)
10-27 21:18:38.188 13649 13649 F DEBUG   :       #32 pc 00000000004a9110  /apex/com.android.runtime/lib64/libart.so!libart.so (offset 0x454000) (art::(anonymous namespace)::InvokeWithArgArray(art::ScopedObjectAccessAlreadyRunnable const&, art::ArtMethod*, art::(anonymous namespace)::ArgArray*, art::JValue*, char const*)+104) (BuildId: 77524e83b39d990eaaf7c2af505c5eae)
10-27 21:18:38.188 13649 13649 F DEBUG   :       #33 pc 00000000004aa460  /apex/com.android.runtime/lib64/libart.so!libart.so (offset 0x454000) (art::InvokeVirtualOrInterfaceWithVarArgs(art::ScopedObjectAccessAlreadyRunnable const&, _jobject*, _jmethodID*, std::__va_list)+424) (BuildId: 77524e83b39d990eaaf7c2af505c5eae)
10-27 21:18:38.188 13649 13649 F DEBUG   :       #34 pc 0000000000387758  /apex/com.android.runtime/lib64/libart.so!libart.so (offset 0x292000) (art::JNI::CallBooleanMethodV(_JNIEnv*, _jobject*, _jmethodID*, std::__va_list)+632) (BuildId: 77524e83b39d990eaaf7c2af505c5eae)
10-27 21:18:38.188 13649 13649 F DEBUG   :       #35 pc 00000000000d2e44  /system/lib64/libandroid_runtime.so (_JNIEnv::CallBooleanMethod(_jobject*, _jmethodID*, ...)+116) (BuildId: f56bc1fc11ff5978e2f1a57f7348acd4)
10-27 21:18:38.188 13649 13649 F DEBUG   :       #36 pc 00000000001482b4  /system/lib64/libandroid_runtime.so (JavaBBinder::onTransact(unsigned int, android::Parcel const&, android::Parcel*, unsigned int)+152) (BuildId: f56bc1fc11ff5978e2f1a57f7348acd4)
10-27 21:18:38.188 13649 13649 F DEBUG   :       #37 pc 000000000004c670  /system/lib64/libbinder.so (android::BBinder::transact(unsigned int, android::Parcel const&, android::Parcel*, unsigned int)+136) (BuildId: 9bf0b9c4a84b3868f2d27d1d2ac1d19f)
10-27 21:18:38.188 13649 13649 F DEBUG   :       #38 pc 000000000005898c  /system/lib64/libbinder.so (android::IPCThreadState::executeCommand(int)+980) (BuildId: 9bf0b9c4a84b3868f2d27d1d2ac1d19f)
10-27 21:18:38.188 13649 13649 F DEBUG   :       #39 pc 0000000000058504  /system/lib64/libbinder.so (android::IPCThreadState::getAndExecuteCommand()+156) (BuildId: 9bf0b9c4a84b3868f2d27d1d2ac1d19f)
10-27 21:18:38.188 13649 13649 F DEBUG   :       #40 pc 0000000000058c40  /system/lib64/libbinder.so (android::IPCThreadState::joinThreadPool(bool)+60) (BuildId: 9bf0b9c4a84b3868f2d27d1d2ac1d19f)
10-27 21:18:38.188 13649 13649 F DEBUG   :       #41 pc 000000000007ee14  /system/lib64/libbinder.so (android::PoolThread::threadLoop()+24) (BuildId: 9bf0b9c4a84b3868f2d27d1d2ac1d19f)
10-27 21:18:38.188 13649 13649 F DEBUG   :       #42 pc 0000000000013638  /system/lib64/libutils.so (android::Thread::_threadLoop(void*)+288) (BuildId: b965af381dbeeb67b1b947b7c61206b3)
10-27 21:18:38.188 13649 13649 F DEBUG   :       #43 pc 00000000000c2794  /system/lib64/libandroid_runtime.so (android::AndroidRuntime::javaThreadShell(void*)+140) (BuildId: f56bc1fc11ff5978e2f1a57f7348acd4)
10-27 21:18:38.188 13649 13649 F DEBUG   :       #44 pc 00000000000e2390  /apex/com.android.runtime/lib64/bionic/libc.so (__pthread_start(void*)+36) (BuildId: bf14cf7a62d1f91755beddd4a937354d)
10-27 21:18:38.188 13649 13649 F DEBUG   :       #45 pc 0000000000083ab0  /apex/com.android.runtime/lib64/bionic/libc.so (__start_thread+64) (BuildId: bf14cf7a62d1f91755beddd4a937354d)
10-27 21:18:39.143  2012  2421 I BootReceiver: Copying /data/tombstones/tombstone_11 to DropBox (system_server_native_crash)
10-27 21:18:39.148  2012  2421 I DropBoxManagerService: add tag=system_server_native_crash isTagEnabled=true flags=0x2
10-27 21:18:39.344  3716 13402 E AndroidRuntime: FATAL EXCEPTION: AsyncTask #22
10-27 21:18:39.344  3716 13402 E AndroidRuntime: Process: com.android.systemui, PID: 3716
10-27 21:18:39.344  3716 13402 E AndroidRuntime: DeadSystemException: The system died; earlier logs will point to the root cause
```

SandHook
```
--------- beginning of crash
10-27 20:27:57.433  2029  4781 F libc    : Fatal signal 11 (SIGSEGV), code 2 (SEGV_ACCERR), fault addr 0x9ab7f920 in tid 4781 (ClientModeImpl), pid 2029 (system_server)
10-27 20:27:57.624 11775 11775 F DEBUG   : *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***
10-27 20:27:57.624 11775 11775 F DEBUG   : LineageOS Version: '17.1-20201027-NIGHTLY-lake'
10-27 20:27:57.624 11775 11775 F DEBUG   : Build fingerprint: 'motorola/lake_retail/lake:10/QPW30.61-18/d18ed:user/release-keys'
10-27 20:27:57.624 11775 11775 F DEBUG   : Revision: 'pvt'
10-27 20:27:57.624 11775 11775 F DEBUG   : ABI: 'arm64'
10-27 20:27:57.625 11775 11775 F DEBUG   : Timestamp: 2020-10-27 20:27:57+0100
10-27 20:27:57.625 11775 11775 F DEBUG   : pid: 2029, tid: 4781, name: ClientModeImpl  >>> system_server <<<
10-27 20:27:57.625 11775 11775 F DEBUG   : uid: 1000
10-27 20:27:57.626 11775 11775 F DEBUG   : signal 11 (SIGSEGV), code 2 (SEGV_ACCERR), fault addr 0x9ab7f920
10-27 20:27:57.626 11775 11775 F DEBUG   :     x0  0000007156e10290  x1  00000000158d9920  x2  0000000016649e30  x3  0000000070de00c8
10-27 20:27:57.626 11775 11775 F DEBUG   :     x4  000000001441c320  x5  0000000000000000  x6  0000000000000000  x7  00000000ffffffff
10-27 20:27:57.626 11775 11775 F DEBUG   :     x8  0000000000000048  x9  000000009ab7f920  x10 00000071b98db4c6  x11 00000071c60d01dc
10-27 20:27:57.626 11775 11775 F DEBUG   :     x12 00000071c60d0224  x13 00000071c60d026c  x14 00000071c60d02cc  x15 0000000000000000
10-27 20:27:57.626 11775 11775 F DEBUG   :     x16 00000071c655a618  x17 0000000000000000  x18 000000711f508000  x19 0000007140824c00
10-27 20:27:57.626 11775 11775 F DEBUG   :     x20 0000000000000000  x21 0000007140824c00  x22 00000071326a6130  x23 00000071b98db4b2
10-27 20:27:57.626 11775 11775 F DEBUG   :     x24 000000000000004c  x25 000000711f824020  x26 0000007140824cb0  x27 000000711f824020
10-27 20:27:57.626 11775 11775 F DEBUG   :     x28 00000071c655e000  x29 000000711f822470
10-27 20:27:57.626 11775 11775 F DEBUG   :     sp  000000711f822410  lr  00000071c60d0338  pc  000000009ab7f920
10-27 20:27:58.253 11775 11775 F DEBUG   : 
10-27 20:27:58.253 11775 11775 F DEBUG   : backtrace:
10-27 20:27:58.253 11775 11775 F DEBUG   :       #00 pc 000000000006b920  /memfd:/jit-cache (deleted)
10-27 20:27:58.253 11775 11775 F DEBUG   :       #01 pc 0000000000136334  /apex/com.android.runtime/lib64/libart.so (art_quick_invoke_stub+548) (BuildId: 77524e83b39d990eaaf7c2af505c5eae)
10-27 20:27:58.253 11775 11775 F DEBUG   :       #02 pc 000000000014506c  /apex/com.android.runtime/lib64/libart.so (art::ArtMethod::Invoke(art::Thread*, unsigned int*, unsigned int, art::JValue*, char const*)+244) (BuildId: 77524e83b39d990eaaf7c2af505c5eae)
10-27 20:27:58.253 11775 11775 F DEBUG   :       #03 pc 00000000004a9110  /apex/com.android.runtime/lib64/libart.so!libart.so (offset 0x454000) (art::(anonymous namespace)::InvokeWithArgArray(art::ScopedObjectAccessAlreadyRunnable const&, art::ArtMethod*, art::(anonymous namespace)::ArgArray*, art::JValue*, char const*)+104) (BuildId: 77524e83b39d990eaaf7c2af505c5eae)
10-27 20:27:58.253 11775 11775 F DEBUG   :       #04 pc 00000000004aab38  /apex/com.android.runtime/lib64/libart.so!libart.so (offset 0x454000) (art::InvokeMethod(art::ScopedObjectAccessAlreadyRunnable const&, _jobject*, _jobject*, _jobject*, unsigned long)+1476) (BuildId: 77524e83b39d990eaaf7c2af505c5eae)
10-27 20:27:58.253 11775 11775 F DEBUG   :       #05 pc 000000000043744c  /apex/com.android.runtime/lib64/libart.so!libart.so (offset 0x3e9000) (art::Method_invoke(_JNIEnv*, _jobject*, _jobject*, _jobjectArray*)+52) (BuildId: 77524e83b39d990eaaf7c2af505c5eae)
10-27 20:27:58.253 11775 11775 F DEBUG   :       #06 pc 00000000000bfc34  /system/framework/arm64/boot.oat (art_jni_trampoline+180) (BuildId: ead7e1b0deb91a66a69e417eff53bea19393fa25)
10-27 20:27:58.253 11775 11775 F DEBUG   :       #07 pc 000000000204f73c  /memfd:/jit-cache (deleted) (com.swift.sandhook.SandHook.callOriginMethod+268)
10-27 20:27:58.253 11775 11775 F DEBUG   :       #08 pc 0000000002060d18  /memfd:/jit-cache (deleted) (com.swift.sandhook.xposedcompat.hookstub.HookStubManager.hookBridge+648)
10-27 20:27:58.253 11775 11775 F DEBUG   :       #09 pc 00000000001365b8  /apex/com.android.runtime/lib64/libart.so (art_quick_invoke_static_stub+568) (BuildId: 77524e83b39d990eaaf7c2af505c5eae)
10-27 20:27:58.253 11775 11775 F DEBUG   :       #10 pc 000000000014508c  /apex/com.android.runtime/lib64/libart.so (art::ArtMethod::Invoke(art::Thread*, unsigned int*, unsigned int, art::JValue*, char const*)+276) (BuildId: 77524e83b39d990eaaf7c2af505c5eae)
10-27 20:27:58.253 11775 11775 F DEBUG   :       #11 pc 00000000002df0d4  /apex/com.android.runtime/lib64/libart.so!libart.so (offset 0x292000) (art::interpreter::ArtInterpreterToCompiledCodeBridge(art::Thread*, art::ArtMethod*, art::ShadowFrame*, unsigned short, art::JValue*)+384) (BuildId: 77524e83b39d990eaaf7c2af505c5eae)
10-27 20:27:58.253 11775 11775 F DEBUG   :       #12 pc 00000000002da3b4  /apex/com.android.runtime/lib64/libart.so!libart.so (offset 0x292000) (bool art::interpreter::DoCall<false, false>(art::ArtMethod*, art::Thread*, art::ShadowFrame&, art::Instruction const*, unsigned short, art::JValue*)+912) (BuildId: 77524e83b39d990eaaf7c2af505c5eae)
10-27 20:27:58.253 11775 11775 F DEBUG   :       #13 pc 000000000059a3f0  /apex/com.android.runtime/lib64/libart.so!libart.so (offset 0x4b3000) (MterpInvokeStatic+368) (BuildId: 77524e83b39d990eaaf7c2af505c5eae)
10-27 20:27:58.253 11775 11775 F DEBUG   :       #14 pc 0000000000130994  /apex/com.android.runtime/lib64/libart.so (mterp_op_invoke_static+20) (BuildId: 77524e83b39d990eaaf7c2af505c5eae)
10-27 20:27:58.253 11775 11775 F DEBUG   :       #15 pc 0000000000000352  [anon:dalvik-classes.dex extracted in memory from /data/user_de/0/android/cache/sandhook/zygote64/hookers/SandHookerNew_3m2g5c8g95gch34dd46g8kpi1n.jar] (SandHookerNew_3m2g5c8g95gch34dd46g8kpi1n.hook+330)
10-27 20:27:58.253 11775 11775 F DEBUG   :       #16 pc 00000000002afd20  /apex/com.android.runtime/lib64/libart.so!libart.so (offset 0x292000) (_ZN3art11interpreterL7ExecuteEPNS_6ThreadERKNS_20CodeItemDataAccessorERNS_11ShadowFrameENS_6JValueEbb.llvm.1271440803783865717+240) (BuildId: 77524e83b39d990eaaf7c2af505c5eae)
10-27 20:27:58.253 11775 11775 F DEBUG   :       #17 pc 0000000000588e8c  /apex/com.android.runtime/lib64/libart.so!libart.so (offset 0x4b3000) (artQuickToInterpreterBridge+1012) (BuildId: 77524e83b39d990eaaf7c2af505c5eae)
10-27 20:27:58.253 11775 11775 F DEBUG   :       #18 pc 000000000013f468  /apex/com.android.runtime/lib64/libart.so (art_quick_to_interpreter_bridge+88) (BuildId: 77524e83b39d990eaaf7c2af505c5eae)
10-27 20:27:58.253 11775 11775 F DEBUG   :       #19 pc 000000000207e968  /memfd:/jit-cache (deleted) (com.android.server.am.ActivityManagerService.broadcastIntent+424)
10-27 20:27:58.253 11775 11775 F DEBUG   :       #20 pc 00000000004e5b5c  /system/framework/arm64/boot-framework.oat (android.app.ContextImpl.sendBroadcastAsUser+428) (BuildId: 268218f903f195e6053515289a297317303f358b)
10-27 20:27:58.253 11775 11775 F DEBUG   :       #21 pc 00000000004e5980  /system/framework/arm64/boot-framework.oat (android.app.ContextImpl.sendBroadcastAsUser+64) (BuildId: 268218f903f195e6053515289a297317303f358b)
10-27 20:27:58.253 11775 11775 F DEBUG   :       #22 pc 00000000001aad2c  /system/framework/oat/arm64/wifi-service.odex (com.android.server.wifi.ClientModeImpl.sendRssiChangeBroadcast+396) (BuildId: c0cf2899ab9881d7ada338e85360c71134703273)
10-27 20:27:58.253 11775 11775 F DEBUG   :       #23 pc 00000000001a5678  /system/framework/oat/arm64/wifi-service.odex (com.android.server.wifi.ClientModeImpl.fetchRssiLinkSpeedAndFrequencyNative+888) (BuildId: c0cf2899ab9881d7ada338e85360c71134703273)
10-27 20:27:58.253 11775 11775 F DEBUG   :       #24 pc 000000000019a350  /system/framework/oat/arm64/wifi-service.odex (com.android.server.wifi.ClientModeImpl$L2ConnectedState.updateLinkLayerStatsRssiAndScoreReportInternal+144) (BuildId: c0cf2899ab9881d7ada338e85360c71134703273)
10-27 20:27:58.253 11775 11775 F DEBUG   :       #25 pc 000000000019c2a0  /system/framework/oat/arm64/wifi-service.odex (com.android.server.wifi.ClientModeImpl$L2ConnectedState.processMessage+5712) (BuildId: c0cf2899ab9881d7ada338e85360c71134703273)
10-27 20:27:58.253 11775 11775 F DEBUG   :       #26 pc 0000000000a7d03c  /system/framework/arm64/boot-framework.oat (com.android.internal.util.StateMachine$SmHandler.processMsg+428) (BuildId: 268218f903f195e6053515289a297317303f358b)
10-27 20:27:58.253 11775 11775 F DEBUG   :       #27 pc 0000000000a7dcdc  /system/framework/arm64/boot-framework.oat (com.android.internal.util.StateMachine$SmHandler.handleMessage+668) (BuildId: 268218f903f195e6053515289a297317303f358b)
10-27 20:27:58.253 11775 11775 F DEBUG   :       #28 pc 0000000000739b94  /system/framework/arm64/boot-framework.oat (android.os.Handler.dispatchMessage+180) (BuildId: 268218f903f195e6053515289a297317303f358b)
10-27 20:27:58.253 11775 11775 F DEBUG   :       #29 pc 000000000073d26c  /system/framework/arm64/boot-framework.oat (android.os.Looper.loop+1756) (BuildId: 268218f903f195e6053515289a297317303f358b)
10-27 20:27:58.254 11775 11775 F DEBUG   :       #30 pc 000000000073bba0  /system/framework/arm64/boot-framework.oat (android.os.HandlerThread.run+544) (BuildId: 268218f903f195e6053515289a297317303f358b)
10-27 20:27:58.254 11775 11775 F DEBUG   :       #31 pc 0000000000136334  /apex/com.android.runtime/lib64/libart.so (art_quick_invoke_stub+548) (BuildId: 77524e83b39d990eaaf7c2af505c5eae)
10-27 20:27:58.254 11775 11775 F DEBUG   :       #32 pc 000000000014506c  /apex/com.android.runtime/lib64/libart.so (art::ArtMethod::Invoke(art::Thread*, unsigned int*, unsigned int, art::JValue*, char const*)+244) (BuildId: 77524e83b39d990eaaf7c2af505c5eae)
10-27 20:27:58.254 11775 11775 F DEBUG   :       #33 pc 00000000004a9110  /apex/com.android.runtime/lib64/libart.so!libart.so (offset 0x454000) (art::(anonymous namespace)::InvokeWithArgArray(art::ScopedObjectAccessAlreadyRunnable const&, art::ArtMethod*, art::(anonymous namespace)::ArgArray*, art::JValue*, char const*)+104) (BuildId: 77524e83b39d990eaaf7c2af505c5eae)
10-27 20:27:58.254 11775 11775 F DEBUG   :       #34 pc 00000000004aa1a4  /apex/com.android.runtime/lib64/libart.so!libart.so (offset 0x454000) (art::InvokeVirtualOrInterfaceWithJValues(art::ScopedObjectAccessAlreadyRunnable const&, _jobject*, _jmethodID*, jvalue const*)+416) (BuildId: 77524e83b39d990eaaf7c2af505c5eae)
10-27 20:27:58.254 11775 11775 F DEBUG   :       #35 pc 00000000004e9f3c  /apex/com.android.runtime/lib64/libart.so!libart.so (offset 0x4b3000) (art::Thread::CreateCallback(void*)+1176) (BuildId: 77524e83b39d990eaaf7c2af505c5eae)
10-27 20:27:58.254 11775 11775 F DEBUG   :       #36 pc 00000000000e2390  /apex/com.android.runtime/lib64/bionic/libc.so (__pthread_start(void*)+36) (BuildId: bf14cf7a62d1f91755beddd4a937354d)
10-27 20:27:58.254 11775 11775 F DEBUG   :       #37 pc 0000000000083ab0  /apex/com.android.runtime/lib64/bionic/libc.so (__start_thread+64) (BuildId: bf14cf7a62d1f91755beddd4a937354d)
```

**Comment by kotori2** [comment]
I think I had this issue only once on Android R. 
```
#00 pc 000000000006b920  /memfd:/jit-cache (deleted)
```
looks like JIT cache corrupted?

**Comment by Tutul-** [comment]
Well the problem seem to have been resolved using the latest v0.5.0.6_4586-master YAHFA.
Doesn't have any crash or unexpected reboot (for now) and my modules seem to work as expected.
